contractcourt/chain_arbitrator.go

```diff
@@ -599,7 +599,12 @@ func (c *ChainArbitrator) ForceCloseContract(chanPoint wire.OutPoint) (*wire.Msg
 	arbitrator, ok := c.activeChannels[chanPoint]
 	c.Unlock()
 	if !ok {
-		return nil, fmt.Errorf("unable to find arbitrator")
+		return nil, errors.New("unable to find arbitrator")
+	}
+
+	if arbitrator.state != StateDefault {
```

Comment:
This check is already done in the `channel arbitrator`. We should instead return the error on the err chan from there I think.

Reply:
Fixed and test added.